brands/amenity/car_sharing.json

```diff
@@ -0,0 +1,48 @@
+{
+  "amenity/car_sharing|Bluely": {
+    "match": [
+      "amenity/car_sharing|Station Bluely"
+    ],
+    "tags": {
+      "amenity": "car_sharing",
+      "brand": "Bluely",
+      "brand:wikidata": "Q16039715",
+      "brand:wikipedia": "fr:Bluely",
+      "name": "Bluely"
+    }
+  },
+  "amenity/car_sharing|Enterprise Car Club": {
+    "match": [
+      "amenity/car_sharing|City Car Club"
+    ],
+    "tags": {
+      "amenity": "car_sharing",
+      "brand": "Enterprise Car Club",
+      "brand:wikidata": "Q5123055",
+      "brand:wikipedia": "en:Enterprise Car Club",
+      "name": "Enterprise Car Club"
+    }
+  },
+  "amenity/car_sharing|Zipcar": {
+    "count": 77,
+    "tags": {
+      "amenity": "car_sharing",
+      "brand": "Zipcar",
+      "brand:wikidata": "Q1069924",
+      "brand:wikipedia": "en:Zipcar",
+      "name": "Zipcar"
+    }
+  },
+  "amenity/car_sharing|stadtmobil": {
+    "match": [
+      "amenity/car_sharing|stadtmobil CarSharing-Station"
+    ],
+    "tags": {
+      "amenity": "car_sharing",
+      "brand": "stadtmobil",
+      "brand:wikidata": "Q2327629",
+      "brand:wikipedia": "en:Stadtmobil",
+      "name": "stadtmobil"
```

Comment:
Should not this name start with a capital letter? And the `brand` too?

Reply:
Based on their signage and website they use an all lowercase name.

Reply:
But I thought I read somewhere that all names in OSM shoudl start with a capital letter. I just can't find the source anymore... Maybe it was also some kind of Osmose error, does anybody else know more?

Reply:
In OSM the name and brand should be as-signed. It's totally ok for them to start with a lowercase letter. Thanks for confirming @tas50